frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -923,6 +936,76 @@ export const experimentLogic = kea<experimentLogicType>([
             actions.closeSecondarySharedMetricModal()
             actions.loadExperiment()
         },
+        createExperimentDashboard: async () => {
+            actions.setIsCreatingExperimentDashboard(true)
+            try {
+                // 1. Create the dashboard
+                // 2. Create secondary metric insights in reverse order
+                // 3. Create primary metric insights in reverse order
+
+                const experimentUrl =
+                    window.location.origin + addProjectIdIfMissing(urls.experiment(values.experimentId))
+                const dashboard: DashboardType = await api.create(
+                    `api/environments/${teamLogic.values.currentTeamId}/dashboards/`,
+                    {
+                        name: values.experiment.name + ' Dashboard',
```

Comment:
It's kind of obvious that it is a "Dashboard". Suggesting this instead:
```suggestion
                        name: 'Experiment: ' + values.experiment.name,
```